src/ActionModal/components/ActionModal.vue

```diff
@@ -705,6 +707,17 @@ export default {
         this.selectedSignMethod
       )
       this.$emit(`txIncluded`)
+      // sending to ga
+      this.sendEvent(
```

Comment:
what exactly does this event track?

Reply:
This event tracks all actions with sending denoms like claiming rewards, sending, deligating, undeligating and redeligating. All actions, related to account amount change

Reply:
should we remove the individual events in each action modal, then?

Reply:
Are we using them right now?

Reply:
yes. we can leave it in for now. no problem.